app/grandchallenge/core/utils/error_messages.py

```diff
@@ -0,0 +1,7 @@
+def format_error_message(error):

```

Comment:
This is specific to validation errors, right? Only there because of the construction of this class in django.

Reply:
Yes. Should I add a check `if isinstance(error, ValidationError)` and error out if not?

Reply:
I think maybe just rename the method so developers know when to use it.